lambda-streaming-ttfb-write-sam-with-dynamodb/example-pattern.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "Lambda Response streaming: Streaming incremental DynamoDB Query results.",
+  "description": "This pattern uses Lambda response streaming to incrementally retrieve and stream DynamoDB results in batches, reducing time-to-first-byte (TTFB) by sending data as it's retrieved.",
```

Comment:
```suggestion
  "description": "This pattern uses AWS Lambda response streaming to incrementally retrieve and stream Amazon DynamoDB results in batches, reducing time-to-first-byte (TTFB) by sending data as it's retrieved.",
```

Reply:
Applied the changes in the latest commit